docs/management/release.rst

```diff
@@ -15,6 +15,13 @@ not include an update to the catalog on Gadi, skip the first two steps below:
       $ export RELEASE=vX.X.X
       $ cd bin
       $ qsub -v version=${RELEASE} build_all.sh
+
+   #. If the `schema <https://github.com/ACCESS-NRI/schema>`_ has changed, or you have not used the intake catalog recently, this step may fail with a *Network is unreachable* error trying to download the schema json files. To workaround::
+      
+      $ module use /g/data/hh5/public/modules/; module load conda/analysis3
```

Comment:
```suggestion
```

Reply:
This will not download the correct version of the schema if it has been changed. Users should load their development environment